### DISCUSSION TITLE
selling the same products by piece ,small box/cartoon and big box/cartoon from a common and unique stock

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#478] [Ideas]
Hi, everyone
I have :(Almost/All supermarkets works like that and needs this method)
100 milk cartoon/box big size ,each cartoon have 20 pieces.
100 ilk cartoon/box  small size , each box have  10 pieces.
all pieces have the same barcode/sku :1000000000000  purchase price : 0.5   sell price 1  (pieces coming from opening small and big box)
big cartoon                      barcode/sku :1000000000001  purchase price : 10   sell price 15
small cartoon                  barcode/sku  :1000000000002  purchase price : 5    sell price  7.5
Default unit purchase : small and big box (we can't purchase it by pieces).
Customers can buy piece, small and big box
Note : I tried "Selling Unit/Determine the unit for sale" Nexopos feature
-Can I create one product with  different variants (3 barcodes/sku) Or I have to create 3 different products?
Any one can explain how to do that ?
-Can I show its stock quantities by these different units in reports sections?
thank you

**Comment by Blair2004** [comment]
Hi, 

Thank you for this.

1- Yes we're planning along with the Raw Material feature, something that will help converting unit. Basically that means if a product is purchased in box (bigger unit), it can be automatically converted into small unit (sale unit). That feature will be added.

2- Regarding variant, it's a scheduled feature as well, we'll go gradually and add those feature, making sure everything works all together.

3- What report are you aiming. Note that from the products list, you can see all available unit per products.
